ci/staging-acceptance-tests.sh

```diff
@@ -0,0 +1,213 @@
+#!/bin/bash
+
+set -uo pipefail
+
+# These are destructive acceptance tests which should only be run in STAGING, there are explicit checks for this, do not override them
+# Requires a user with cloud_controller.admin access to run since it will be creating/deleting users, organizations and org quotas
+# These are destructive to the `sandbox-fedramp`.  This org was chosen since it: wasn't in use, we have email access to the domain, should remain in the CSV file maintaining the list of verified gov agencies
+
+# Log into CF as an admin
+echo "Logging into ${CF_API} as user ${CF_ADMIN_USER}..."
+cf login -a ${CF_API} -u ${CF_ADMIN_USER} -p "${CF_ADMIN_PASSWORD}" -o cloud-gov -s bots >/dev/null 2>&1
+
+# Confirm that we're targeting the correct API
+api_target=$(cf api | grep -i 'API endpoint' | awk '{print $3}')
+if [[ "$api_target" != *"fr-stage"* ]]; then
+  echo "### THIS IS A DESTRUCTIVE TEST to the sandbox-fedramp org, DO NOT RUN IN PRODUCTION ###"
+  echo "Error: Not targeting staging. Current API endpoint: $api_target, exiting for your own safety."
+  exit 1
+fi
+
+# Create a random 32-character password with hyphens
+PASSWORD=$(cat /dev/urandom | base64 | tr -dc '0-9a-zA-Z' | head -c32)
+
+# Create the user
+echo "Creating CF user test.user@fedramp.gov with a random password..."
+cf create-user "test.user@fedramp.gov" "$PASSWORD" >/dev/null 2>&1
+
+# Sleep for 45 seconds, app runs every 30 seconds
+echo "Sleeping test for 45 seconds to wait for sandbox-bot app to process the new user..."
+sleep 45
+
+# Observe the output from the app, it should create a new org, space and quotas within 30 seconds
+ORG_NAME="sandbox-fedramp"
+SPACE_NAME="test.user"
+EXPECTED_ORG_QUOTA="sandbox-fedramp"
+EXPECTED_SPACE_QUOTA="sandbox_quota"
+REQUIRED_SECURITY_GROUPS=("public_networks_egress" "trusted_local_networks_egress")
+
+echo "🔍 Checking for org '$ORG_NAME'..."
+if cf orgs | grep -q "^$ORG_NAME$"; then
+  echo "✅ Org '$ORG_NAME' found. Targeting..."
+  cf target -o "$ORG_NAME" >/dev/null 2>&1
+else
+  echo "❌ Org '$ORG_NAME' not found."
+fi
+
+echo "🔍 Checking org quota for '$ORG_NAME'..."
+org_quota=$(cf org "$ORG_NAME" | awk -F': ' '/quota:/ {print $2}' | xargs || echo "UNKNOWN")
```

Comment:
```suggestion
org_quota=$(cf org "$ORG_NAME" | awk -F': ' '/quota:/ {print $2}' | tr -d '[:space:]' || echo "UNKNOWN")
```

Using `xargs` to trim whitespace is clever, but I somewhat prefer `tr -d '[:space:]' because it is somewhat more obvious what the operation is doing.

Reply:
I've switched to using `tr -d '[:space:]'` for trimming all the white space instead of just leading and trailing whitespace with `xargs`